internal/services/appservice/helpers/web_app_schema.go

```diff
@@ -297,12 +296,6 @@ func SiteConfigSchemaWindows() *pluginsdk.Schema {
 					Description: "Should all outbound traffic to have Virtual Network Security Groups and User Defined Routes applied? Defaults to `false`.",
 				},
 
-				"auto_swap_slot_name": {
```

Comment:
Is there anyway someone was using this value? Just want to make sure we don't break anyone with it's removal

Reply:
It's meaningless on this resources, just an accidental carry over from the way the old resources were designed, so totally safe to remove.